components/symbol/src/Symbol.js

```diff
@@ -0,0 +1,115 @@
+/** @jsx jsx */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { jsx, useTheme, paint } from '@westpac/core';
+
+// ==============================
+// Utils
+// ==============================
+
+const asArray = val => (Array.isArray(val) ? val : [val]);
+const round = f => Math.round(f * 100) / 100; //2DP
```

Comment:
is this necessary? what is the convention/best practise for rounding css values

Reply:
Very hard to find a definitive answer 🤔

The rounding of fractional pixel values is not consistent across browsers. Some will round down (floor), some up (ceil) and some truncate decimal places.

My thinking was we should set it as accurate as possible; let the browser decide, but 1DP is probably all that's required if we do keep decimal places.

We can probably just use Math.round() in this case to return a 0DP pixel value. This will skew resized symbols/logos ever so slightly; maybe only noticeable on retina monitors, if at all.

@florettewest have you had to deal with this previously?

Reply:
@jonnystening, I am for rounding to 1 or 2DP. Because it is more accurate, and that's the whole point of using SVGs. Browsers might align in the future?